darts/timeseries.py

```diff
@@ -257,20 +248,25 @@ def __init__(self, xa: xr.DataArray):
                 include=np.number, exclude=self.dtype
             ).columns
 
-            changes = {col: self.dtype for col in cols_to_cast}
             # Calling astype is costly even when there's no change...
-            if len(changes) > 0:
-                static_covariates = static_covariates.astype(changes, copy=False)
+            if not cols_to_cast.empty:
```

Comment:
you cannot just write `if cols_to_cast` ?

Reply:
no, since it's a pandas Series, and it complains about:

```
ValueError: The truth value of a Series is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().
```